io.github.masterbillwoo.rise_of_the_penguins_gb.yml

```diff
@@ -0,0 +1,28 @@
+app-id: io.github.masterbillwoo.rise_of_the_penguins_gb
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+base: org.godotengine.godot.BaseApp
+base-version: '4.4.1'
+sdk: org.freedesktop.Sdk
+command: godot-runner
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=all
+
+modules:
+  - name: rise-of-the-penguins-gb
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://raw.githubusercontent.com/masterbillwoo/rise-of-the-penguins-gb-linux-pck-export/main/rise%20of%20the%20penguins%20gb.pck
```

Comment:
you should never reference a file from the tip of a branch. This will break because its not immutable.